db/structure.sql

```diff
@@ -472,90 +474,6 @@ CREATE TABLE public.ar_internal_metadata (
 );
 
 
---
--- Name: building_contacts_id_seq; Type: SEQUENCE; Schema: public; Owner: -
```

Comment:
should this be present? review..

Reply:
for what it's worth these are also not present in my local db (and get removed from the structure file by `rails db:migrate` when I run that locally)